### ISSUE TITLE
A11y: Insufficient color contrast for Image Gallery pagination/arrows

### ISSUE THREAD

**Issue by shopwareBot** [shopware/platform#6311]
The inactive pagination dots and image gallery arrows have insufficient contrast. Recommend to change to {{rgb(74,84,91)}}

**Comment by shopwareBot** [comment]
[public] Shopware automated response: This issue is linked to the internal issue: **NEXT-40361**



[created from NEXT-40361, comment 534804]

**Comment by shopwareBot** [comment]
[public] 

 > Priority changed to: **Low**

 Thank you for your report. After careful consideration, we've concluded that this issue does not require immediate attention or isn't something we can address right now. We prioritize openness and clear communication with our community, so we want to be transparent. We may not resolve all reported issues, and this one might remain unresolved. To stay effective, we periodically clean our backlog, and this issue could be closed in the future.

 For now, we'll keep this ticket open to invite further feedback from the community in case we've underestimated its impact and to allow us to revisit it later.



[created from NEXT-40361, comment 538722]
